### PR TITLE
Catch exception from has_symbol

### PR DESCRIPTION
**Pull request by Lobotuerk**
Signed-off-by: lobotuerk <jtlorente@ekumenlabs.com>

### CODE REVIEW

File: rosidl_typesupport_c/src/type_support_dispatch.hpp [hunk 1]
```diff
@@ -90,12 +90,22 @@ get_typesupport_handle_function(
       }
       auto clib = static_cast<const rcpputils::SharedLibrary *>(map->data[i]);
       lib = const_cast<rcpputils::SharedLibrary *>(clib);
-      if (!lib->has_symbol(map->symbol_name[i])) {
+
+      void * sym;
```
Comment (hidmic) [inline rosidl_typesupport_c/src/type_support_dispatch.hpp:94]: @Lobotuerk nit: 
```suggestion
      void * sym = nullptr;
```

**Review by hidmic — APPROVED** [review]
LGTM pending green CI

File: rosidl_typesupport_c/src/type_support_dispatch.hpp (hunk 1 above)
Comment (Lobotuerk) [reply rosidl_typesupport_c/src/type_support_dispatch.hpp]: Updated on 506e573

**Comment by Lobotuerk** [comment]
CI up to `rcl` and `rosidl_typesupport`:
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/12561/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/7529/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/10272/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/12483/)

**Review by hidmic — APPROVED** [review]

**Review by Blast545 — APPROVED** [review]
LGTM. I'd change the title to be "Catch exception from has_symbol", IMO the current title is a bit misleading.

**Comment by Lobotuerk** [comment]
Merging after green CI

**Pull request merged**